metricbeat/mb/event.go

```diff
@@ -109,27 +109,26 @@ func (e *Event) BeatEvent(module, metricSet string, modifiers ...EventModifier)
 //       "rtt": 115
 //     }
 func AddMetricSetInfo(module, metricset string, event *Event) {
-	info := common.MapStr{
-		"name":   metricset,
-		"module": module,
+	e := common.MapStr{
+		"event": common.MapStr{
+			"dataset": metricset,
+			"module":  module,
+		},
 	}
 	if event.Host != "" {
-		info["host"] = event.Host
+		e.Put("service.hostname", event.Host)
```

Comment:
@webmat To make things a bit more interesting, `Host` here contains also the Port number. For example `elasticsearch:9200`.